hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/management/JpaDistributionSetManagement.java

```diff
@@ -786,7 +786,7 @@ public DistributionSet getValid(final long id) {
     public void invalidate(final DistributionSet distributionSet) {
         final JpaDistributionSet jpaSet = (JpaDistributionSet) distributionSet;
         jpaSet.invalidate();
-        distributionSetRepository.save(AccessController.Operation.DELETE, jpaSet);
+        distributionSetRepository.save(jpaSet);
```

Comment:
Since DistributionSetManagement#findAll will still return the invalidated entities, I decided to keep the UPDATE permissions as required:

    @Override
    public Slice<DistributionSet> findAll(final Pageable pageable) {
        return JpaManagementHelper.findAllWithoutCountBySpec(distributionSetRepository, pageable, List.of(
                DistributionSetSpecification.isNotDeleted()));
    }

It was already defined in the DistributionSetManagement#invalidate as well:

    @PreAuthorize(SpringEvalExpressions.HAS_AUTH_UPDATE_REPOSITORY)
    void invalidate(DistributionSet distributionSet);